gradle/coverage.gradle

```diff
@@ -3,6 +3,8 @@ jacoco {
 }
 
 jacocoTestReport {
+    executionData "${project.buildDir}/jacoco/test.exec", "${project.buildDir}/jacoco/functionalTest.exec"
```

Comment:
I don't see where functionalTest.exec is being used- was there a file accidentally excluded from the commit?